towhee/hub/repo_manager.py

```diff
@@ -40,6 +38,8 @@ class RepoManager:
             The author of the repo.
         repo (`str`):
             The name of the repo.
+        root (`str`):
```

Comment:
Can you ignore this file, and there is another PR #484  for it.

Reply:
We can just leave it here and keep your change when resolve conflicts

Reply:
OK